### PR TITLE
fix(deps): library support for angular 13

### PR DESCRIPTION
**Pull request by atlawrie**
Updates the library and its dependencies to use Angular 13. Addresses the current need when installing the library with node to use --legacy-peer-deps.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -28,16 +28,6 @@
...
-        },
-        "lint": {
-          "builder": "@angular-devkit/build-angular:tslint",
-          "options": {
-            "tsConfig": [
-              "projects/imgix-angular/tsconfig.lib.json",
-              "projects/imgix-angular/tsconfig.spec.json"
-            ],
-            "exclude": ["**/node_modules/**"]
-          }
```
Comment (luqven) [inline angular.json:40]: I'm OOL, why do we need to remove the `lint` here?

File: projects/imgix-angular-example/src/test.ts [hunk 2]
```diff
@@ -12,7 +12,9 @@ declare const require: any;
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
-  platformBrowserDynamicTesting(),
+  platformBrowserDynamicTesting(), {
+    teardown: { destroyAfterEach: false }
```
Comment (luqven) [inline projects/imgix-angular-example/src/test.ts:16]: Same as above, I'm Angular ignorant, but for posterity, what does setting this to `false` solve?

**Review by luqven — COMMENTED** [review]
I'm very OOL on angular but assuming all tests pass and no issues testing locally, LGTM

michael-buble-check-gif-by-bubly

File: projects/imgix-angular/package.json [hunk 3]
```diff
@@ -39,8 +39,8 @@
     "urijs": "1.19.8"
   },
   "peerDependencies": {
-    "@angular/common": "8.x || 9.x || 10.x || 11.x || 12.x",
-    "@angular/core": "8.x || 9.x || 10.x || 11.x || 12.x",
-    "@angular/platform-browser": "8.x || 9.x || 10.x || 11.x || 12.x"
+    "@angular/common": "8.x || 9.x || 10.x || 11.x || 12.x || 13.x",
+    "@angular/core": "8.x || 9.x || 10.x || 11.x || 12.x || 13.x",
+    "@angular/platform-browser": "8.x || 9.x || 10.x || 11.x || 12.x || 13.x"
```
Comment (sherwinski) [inline projects/imgix-angular/package.json:44]: Out of curiosity, do we know if these changes allow us to support angular 14 as well?

File: angular.json (hunk 1 above)
Comment (atlawrie) [reply angular.json:40]: TSLint was deprecated in Angular 11 and removed in Angular 13. See https://github.com/angular/angular-cli/issues/20105

File: projects/imgix-angular-example/src/test.ts (hunk 2 above)
Comment (atlawrie) [reply projects/imgix-angular-example/src/test.ts:16]: Angular 13 sets the `teardown` flag to `true` by default for `TestBed`; for compatibility reasons, during the Angular 12 to 13 update process `ng update` opts out of the new teardown behavior.

File: projects/imgix-angular/package.json (hunk 3 above)
Comment (atlawrie) [reply projects/imgix-angular/package.json:44]: Resolved offline.

**Review by luqven — APPROVED** [review]
do-it-episode-3-gif-by-star-wars

**Pull request merged**